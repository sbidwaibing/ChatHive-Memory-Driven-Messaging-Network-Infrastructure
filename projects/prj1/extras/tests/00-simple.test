+ @zdu test #xx
This is a test message
.
? test 
.
? test #xx
.
