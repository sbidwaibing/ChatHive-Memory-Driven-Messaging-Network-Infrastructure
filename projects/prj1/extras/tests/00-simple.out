@zdu test #xx
This is a test message
@zdu test #xx
This is a test message
